.github/workflows/update_v8_snapshot_cache.yml

```diff
@@ -1,10 +1,136 @@
 name: Update V8 Snapshot Cache
-on: [workflow_dispatch]
+on:
+  schedule:
+    # Run every Wednesday at 00:00 UTC
+    - cron: '0 0 * * 3'
+  push:
+    branches:
+      - ryanm/feature/v8-snapshots-auto-pr
```

Comment:
Just to check is this a scheduled PIPELINE or a scheduled WORKFLOW? The reason I ask https://circleci.com/docs/configuration-reference/#schedule

> The scheduled workflows feature is set to be deprecated. Using scheduled pipeline. 

This is in a `workflows` file and called `schedule` - just want to verify we are not using a soon-to-be-deprecated feature.

Reply:
Ah yeah. That's circle ci. I'm using it in a github action where it is not deprecated.